src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/HostBuilderTests.cs

```diff
@@ -560,6 +560,42 @@ public void ValidateOnBuildEnabledInDevelopment()
             Assert.Throws<AggregateException>(() => hostBuilder.Build());
         }
 
+        [Fact]
+        public void ScopeValidationNotEnabledInDevelopmentWithServiceProviderChanges()
+        {
+            using var host = new HostBuilder()
+                .UseEnvironment(Environments.Development)
+                .ConfigureServices(serices =>
```

Comment:
nit: "serices" -> "services" (other tests here have the same problem, so I supposed we can leave as-is)

Reply:
Ick, yeah, lets fix later at once.